lib/semian/sysv_state.rb

```diff
@@ -0,0 +1,31 @@
+require 'forwardable'
+
+module Semian
+  module SysV
+    class State < Semian::Simple::State #:nodoc:
+      include SysVSharedMemory
+      extend Forwardable
+
+      SYM_TO_NUM = {closed: 0, open: 1, half_open: 2}.freeze
+      NUM_TO_SYM = SYM_TO_NUM.invert.freeze
```

Comment:
Simple has the lookup hard-coded, as requested earlier. If other drivers want them, why wouldn't then just use `Semian::SysV::State::SYM_TO_NUM`, etc?